sqlite/src/build.c

```diff
@@ -5511,21 +5519,29 @@ char *sqlite3DescribeIndexOrder(
      }
      isExpr = 1;
   }
-  ret = sqlite3_mprintf("(%s)%s",
-    isExpr?pExprDesc:pTbl->aCol[pIdx->aiColumn[0]].zName, pDesc);
-
+  if( isExpr ){
+    ret = sqlite3_mprintf("(%s) %s", pExprDesc, pDesc);
```

Comment:
Not sure - Dorin can comment - are pExprDesc already correctly quoted? (in this routine and other similar uses)